### ISSUE TITLE
not working at all iPhone 3Gs 4.3.3 (1000 contacts, 1/3 with pictures)

### ISSUE THREAD

**Issue by ohadhawk** [ClockworkMod/DeskSMS#71]
I just installed the app. 
On the website when I check the connection it says "true".
On the iPhone, if only "DeskSMS“ is ON, then there are no crashes (but it also doesn't get synced to anything)
Once I turn on one of the other options (push to Gmail, Gtalk or web), after about a minute I see the iPhone RAM gradually decreasing till it reaches the bare minimum (6Mb) and the the iPhone resprings.
It keeps this cycle forever.
While it's happening I was also trying the check the app usage by texting to the iPhone. It never arrived to gmail/gtalk/web.
I tried sending a text from the web to my phone number (should have send a text message from the phone to itself). Although on the web it seems successful, on the phone no message was actually received.
So basically the app has never worked for me.
I tried the enabling every sync option separately, but it also didn't help.

I don't know if the app start by initially syncing the contacts on first use, that would explain the usage of memory.
I have about 1000 contacts, of which about 1/3 have pictures.
my AddressBook.sqlitedb is about 1Mb,
my AddressBookImages.sqlitedb is about _350MB_.

I will supply whatever debugging info you need.
